Kickstarter-iOS/Views/AmountInputView.swift

```diff
@@ -5,8 +5,8 @@ import UIKit
 class AmountInputView: UIView {
   // MARK: - Properties
 
-  private lazy var label: UILabel = { UILabel(frame: .zero) }()
-  private lazy var textField: UITextField = { UITextField(frame: .zero) }()
+  private(set) lazy var label: UILabel = { UILabel(frame: .zero) }()
```

Comment:
Exposing these in order for us to being able to bind these with `rac` extensions